src/pcb/pcb_silkscreen_oval.ts

```diff
@@ -0,0 +1,19 @@
+import { z } from "zod"
+import { distance } from "../units"
+import { layer_ref, visible_layer } from "./properties/layer_ref"
+import { point } from "src/common/point"
+
+export const pcb_silkscreen_oval = z
+  .object({
+    type: z.literal("pcb_silkscreen_oval"),
+    pcb_silkscreen_oval_id: z.string(),
+    pcb_component_id: z.string(),
+    center: point,
+    radiusX: distance,
+    radiusY: distance,
```

Comment:
```suggestion
    radius_x: distance,
    radius_y: distance,
```

Convention for soup spec is all underscore (other modules are generally camel case, the soup is special because it is supposed to be sent to a database)

Reply:
Okay